src/odemis/gui/model/dye.py

```diff
@@ -45,8 +45,8 @@ def _clean_up_name(name):
     name = name.strip()
 
     # Some names are HTML escaped, we could use HTMLParser().unescape, but for
-    # now, we keep it simple, as there is only one character escaped
-    name = name.replace("&#39;", "'")
+    # now, we keep it simple, as there are only a few characters escaped.
+    name = name.replace("&#39;", "'").replace("&amp;", "&").replace("&quot;", "\"")
```

Comment:
Is this really simpler than:
>		parser = html.parser.HTMLParser()
>		name = (parser.unescape(text))
		
Or is using the HTMLParser more complicated than this?

Reply:
The problem is that I don't know if all the database is actually properly encoded in HTML. Especially, the database is regularly updated. It might work now, but not in the future. So we need to handle the potential case when unescaping HTML fails. Probably just another 3 easy lines though...
Let's switch to it when the current code doesn't fit on a single line anymore :-D

Reply:
That sounds fair ;)